### PR TITLE
Солуянов - Лабораторная работа №3

### PR DESCRIPTION
**Pull request by alexsoll**

### CODE REVIEW

**Comment by alexsoll** [comment]
@VadimKutovoi @EOKruglov code review please

File: modules/vector-3d/application/vector3d.cpp [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+#include <stdlib.h>
+#include <stdio.h>
+#include <string>
+
+#include "include/vector-3d-calculator.h"
+
+int main(int argc, const char** argv) {
+    VectorCalculator app;
+    std::string output = app(argc, argv);
+    printf("%s\n", output.c_str());
```
Comment (VadimKutovoi) [inline modules/vector-3d/application/vector3d.cpp]: Why not "cout"?

File: modules/vector-3d/src/vector_3d.cpp [hunk 2]
```diff
@@ -141,3 +140,4 @@ bool Vector3d::operator == (const Vector3d& v) const {
 bool Vector3d::operator != (const Vector3d& v) const {
     return !(*this == v);
 }
+
```
Comment (VadimKutovoi) [inline modules/vector-3d/src/vector_3d.cpp]: You removed and added lines not in your file

**Comment by alexsoll** [comment]
@alexbatashev Please review

File: modules/vector-3d/CTestTests.txt [hunk 3]
```diff
@@ -0,0 +1,87 @@
+#############################################
+##### Testing
+#############################################
+
+set(prefix "${MODULE}")
+
+add_test(
+    NAME ${prefix}_can_Run
```
Comment (alexbatashev) [inline modules/vector-3d/CTestTests.txt]:
```suggestion
    NAME ${prefix}_Can_Run
```

File: modules/vector-3d/CTestTests.txt [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    COMMAND ${APPLICATION}
+)
+
+set_tests_properties (${prefix}_can_Run PROPERTIES
```
Comment (alexbatashev) [inline modules/vector-3d/CTestTests.txt]:
```suggestion
set_tests_properties (${prefix}_Can_Run PROPERTIES
```

File: modules/vector-3d/application/vector3d.cpp (hunk 1 above)
Comment (alexbatashev) [reply modules/vector-3d/application/vector3d.cpp]: @VadimKutovoi sounds reasonable
@alexsoll why?

**Pull request merged**